eng/build.yml

```diff
@@ -214,7 +214,7 @@ jobs:
     # Publish test results to Azure Pipelines
     - task: PublishTestResults@2
       inputs:
-        testResultsFormat: xUnit
+        testResultsFormat: VSTest
```

Comment:
Will vstest still give us the results in html files too?

Reply:
No. `dotnet test` right now has TRX output, text output, and sequence points (xml list) to tell you what tests have run.

Reply:
That's unfortunate. I really like to download the .html file with a click or two and get the test's logging (at least SOS's) in a nicely formatted web page.

Reply:
I just tried to look at the failing MacOS legs below and the xml file is totally unreadable.  Is there something I can use to format it or view it better?

Reply:
I think I can add this.